src/sortStudents.ts

```diff
@@ -1,16 +1,61 @@
+/* eslint-disable max-len */
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const getAverageGrade = (grades: number[]): number => {
+  return grades
+    .reduce((sum, grade) => sum + grade, 0) / grades.length;
+};
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  sortorder: SortOrder,
+): Student[] {
+  const copyStudents: Student[] = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      return sortorder === 'asc'
+        ? copyStudents.sort((first, second) => first[sortBy]
```

Comment:
not obvious name, what is `first and second` 
```suggestion
        ? copyStudents.sort((firstStudent, secondStudent) => first[sortBy]
```